### PR TITLE
Update readme to point to the process of reporting a vulnerability

### LINKED ISSUE #3246: Security reporting process is hard to find
The main page at https://github.com/openenclave/openenclave has a Send Feedback section that only mentions the mailing list and filing github issues, implying that one should file a github issue or send a public email to report a security issue.

The actual process is buried down at https://github.com/openenclave/openenclave/blob/master/docs/Community/Contributing.md#reporting-security-issues which you'd only discover by trying to learn how to generate a fix, since the only path to get to that page is by looking for instructions on how to contribute to the SDK like opening pull requests, etc.

Instead I suggest the security reporting process should be moved to the main page under Send Feedback.

### PR DESCRIPTION
**Pull request by radhikaj**
Fixes #3246 
Signed-off-by: Radhika Jandhyala <radhikaj@microsoft.com>

### CODE REVIEW

File: docs/Community/Contributing.md [hunk 1]
```diff
@@ -19,13 +19,7 @@ is defined in the documents linked below:
...
 -------------------------
 
-Security issues and bugs should be reported privately, via email, to the
-Microsoft Security Response Center (MSRC) at <secure@microsoft.com>. You should
-receive a response within 24 hours. If for some reason you do not, please follow
-up via email to ensure we received your original message. Further information,
-including the [MSRC PGP](https://technet.microsoft.com/en-us/security/dn606155)
-key, can be found in the [Security
-TechCenter](https://technet.microsoft.com/en-us/security/default).
+To report a security issue, please follow the [process to report a vulnerability](/security.md#reporting-a-vulnerability).
```
Comment (dthaler) [inline docs/Community/Contributing.md:22]: the file is all upper case SECURITY.md
```suggestion
To report a security issue, please follow the [process to report a vulnerability](/SECURITY.md#reporting-a-vulnerability).
```

File: README.md [hunk 2]
```diff
@@ -81,3 +81,5 @@ Send general questions, announcements, and discussion to the
 
 To report a problem or suggest a new feature, file a
 [GitHub issue](https://github.com/openenclave/openenclave/issues).
+
+To report a security issue, please follow the [process to report a vulnerability](security.md#reporting-a-vulnerability).
```
Comment (mingweishih) [inline README.md:85]:
```suggestion
To report a security issue, please follow the [process to report a vulnerability](SECURITY.md#reporting-a-vulnerability).
```

**Review by mingweishih — APPROVED** [review]

**Review by dthaler — APPROVED** [review]

**Review by CodeMonkeyLeet — APPROVED** [review]
LGTM, thanks for addressing this!

**Pull request merged**